test_scripts/AppServices/commonAppServices.lua

```diff
@@ -191,4 +191,15 @@ function commonAppServices.getAppServiceID(app_id)
   return serviceIDs[app_id]
 end
 
+--[[Timeout]]
+function commonAppServices.getRpcPassThroughTimeoutFromINI()
```

Comment:
Better practice would be to set the RPC Pass Through timeout directly with `common.setSDLIniParameter` and use that value rather than reading the default value.

Reply:
Small detail, will still approve without